data/remnant outfits.txt

```diff
@@ -449,6 +449,18 @@ outfit "Tuning Rifle"
 	"unplunderable" 1
 	description "The Tuning Rifle uses sound waves to destabilize the weaker internal tissue of many races and liquefies them."
 
+outfit "Void Rifle"
+	category "Special"
+	licenses
+		Remnant
+	cost 1000000
+	thumbnail "outfit/void rifle"
+	"capture attack" 20
+	"capture defense" 20
+	"unplunderable" 1
+	description "How the Void Rifle works isn't clearly understood, but when fired it emits some kind of non-visible radition that has a strong paralyzing effect on the nervous system. Given that it is an area effect that penetrates walls and other solid objects, it is an incredibly effective weapon."
```

Comment:
> description "How the Void Rifle works isn't clearly understood, but when fired it emits some kind of non-visible radition that has a strong paralyzing effect on the nervous system. Given that it is an area effect that penetrates walls and other solid objects, it is an incredibly effective weapon."

it emits a kind of non-visible

Given that it has an area of effect that

Reply:
Thanks. Fixed!